src/script/js/claimMessageModule.js

```diff
@@ -0,0 +1,47 @@
+import { ethers } from 'ethers';
+import { LineaSDK } from '@consensys/linea-sdk';
+import dotenv from 'dotenv';
+
+dotenv.config(); // Load environment variables
+
+// Securely read the private key from the environment variable
+const privateKey = process.env.L2_SIGNER_PRIVATE_KEY;
+if (!privateKey) {
+    throw new Error('Private key not found in environment variables');
+}
+
+// Initialize the Linea SDK
+const lineaSDK = new LineaSDK({
+  l2RpcUrl: process.env.L2_RPC_URL ?? "",
+  l2SignerPrivateKey: process.env.L2_SIGNER_PRIVATE_KEY ?? "",
```

Comment:
I believe that `privateKey` variable could be used directly because it's already tested for emptiness above
```suggestion
  l2SignerPrivateKey: privateKey,
```